packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -399,6 +400,17 @@ export function getUsersWithCredentialsConsideringContactOwner({
   return contactOwnerAndFixedHosts;
 }
 
+const getStartTime = (
+  startTimeInput: string,
+  timeZone?: string,
+  eventType: Exclude<Awaited<ReturnType<typeof getRegularOrDynamicEventType>>, null>
```

Comment:
Can we be more specific in the typing here? just specify minimumBookingNotice as part of the event type.